README.md

```diff
@@ -1,21 +1,15 @@
 # Acrolinx Sidebar Java SDK
 
-[![Maven Central](https://img.shields.io/maven-central/v/com.acrolinx.client/sidebar-sdk)](https://search.maven.org/artifact/com.acrolinx.client/sidebar-sdk)
```

Comment:
I believe this badge is valuable. It has the URL to Maven Central. Let me know what you think.

Reply:
The URL to Maven Central appears twice (a bit further down) in this README file.